pypesto/visualize/misc.py

```diff
@@ -225,3 +225,57 @@ def process_start_indices(start_indices: Union[int, Iterable[int]],
                      start_index < max_length]
 
     return start_indices
+
+
+LEN_RGB = 3
+LEN_RGBA = 4
+TYPING_RGB = Tuple[(float,) * LEN_RGB]
+TYPING_RGBA = Tuple[(float,) * LEN_RGBA]
+TYPING_RGB_RGBA = Union[TYPING_RGB, TYPING_RGBA]
+RGBA_MIN = 0  # min value for an RGBA element
+RGBA_MAX = 1  # max value for an RGBA element
+RGBA_ALPHA = 3  # zero-indexed fourth element in RGBA
+RGBA_WHITE = (RGBA_MAX, RGBA_MAX, RGBA_MAX, RGBA_MAX)
+RGBA_BLACK = (RGBA_MIN, RGBA_MIN, RGBA_MIN, RGBA_MAX)
```

Comment:
I thought we had something similar in the waterfall visualization routine... Unify?

Reply:
Unified into a "constants.py" file.

Reply:
töp